website/docs/components/template/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Template
+hidden: true
```

Comment:
```suggestion
hidden: false
```

I think it would be useful temporarily to have this visible so we can see it in the website and use it to reference as we go about our updates. what do you think?

Reply:
Good call! I'll get it updated.